seed/static/seed/js/services/inventory_service.js

```diff
@@ -498,6 +498,37 @@ angular.module('BE.seed.service.inventory', []).factory('inventory_service', [
       localStorage.setItem('detailProfiles.' + key, JSON.stringify(profiles));
     };
 
+    inventory_service.get_property_column_names_for_org = function (org_id) {
+      return $http.get('/api/v3/columns/', {
+        params: {
+          inventory_type: 'property',
+          organization_id: org_id,
+          only_used: false,
+          display_units: true
+        }
+      }).then(function (response) {
+        let property_columns = response.data.columns.filter(column => column.table_name == 'PropertyState');
+        return property_columns.map(a => {
+          return { 'column_name': a.column_name, 'display_name': property_columns.find(x => x.column_name == a.column_name).display_name };
```

Comment:
```suggestion
          return { 'column_name': a.column_name, 'display_name': a.display_name };
```
Would this work? Same goes for taxlot func

Reply:
Actually, now that we are intelligently removing the duplicates, yeah, that would work much nicer.